decidim-api/lib/decidim/api.rb

```diff
@@ -7,6 +7,23 @@ module Decidim
   # This module holds all business logic related to exposing a Public API for
   # decidim.
   module Api
+    include ActiveSupport::Configurable
+
+    # defines the schema max_per_page to configure GraphQL pagination
+    config_accessor :schema_max_per_page do
+      50
+    end
+
+    # defines the schema max_per_page to configure GraphQL query complexity
```

Comment:
max_per_page?

Reply:
copy/paste programming :spaghetti:

Reply:
# defines the schema **max_complexity** to configure GraphQL query complexity